Add capabilities/platform documentation (#3542)
meilisearch/documentation: capabilities/platform/infrastructure/regions.mdx

---
title: Cloud regions
description: Reference of all available Meilisearch Cloud regions with location and latency guidance.
---

Meilisearch Cloud is available in six regions. Choosing the right region is one of the most impactful decisions for search latency: network round-trip time between your users and the Meilisearch instance is typically the largest contributor to perceived search speed.

## Available regions

| Code | Location | Recommended use case |
|------|----------|---------------------|

> | `FRA` | Frankfurt, Germany | European users (GDPR-compliant) |

Author: Fixed: FRA now reads "EU data residency" and the note no longer asserts guaranteed GDPR compliance.

| `LON` | London, United Kingdom | UK and Western Europe |
| `SGP` | Singapore | Southeast Asia, Oceania, and APAC |
| `JPN` | Japan | Japan and Northeast Asia |
| `SFO` | San Francisco, United States | US West Coast and Pacific |

> | `NYC` | New York, United States | US East Coast, Latin America, and global default |

Reviewer: We are missing Paris, Sydney, and Sao Paulo

Author: Added: Paris (PAR), Sydney (SYD), and São Paulo (SAO) are now in the regions table, with Paris noted as low-carbon on the AWS European Sovereign Cloud.

## Choosing a region

**Minimize latency for your users.** A search that takes 5 ms to execute on the server can feel instant or sluggish depending on whether the user is 10 ms or 200 ms away. Choose the region closest to the majority of your users.

> **Consider data residency requirements.** If your application handles personal data subject to GDPR or other regional regulations, ensure your project is hosted in a compliant region. `FRA` and `LON` are located within the EU and UK.

Reviewer: ```suggestion
**Consider data residency requirements.** If your application handles personal data subject to GDPR or other regional regulations, ensure your project is hosted in a compliant region. `FRA` is located in the EU, while `LON` is located in the UK.
```

Author: Applied, keeping PAR: "FRA and PAR are located in the EU, while LON is located in the UK."

**Plan for multiple regions if you have a global audience.** You can create separate projects per region and route search traffic to the nearest instance from your application layer. Each project is billed independently.

## Changing a region

Regions cannot be changed after a project is created. To move to a different region, create a new project in the target region and re-index your documents.

## Requesting a new region

> The list of available regions grows over time. If none of the current regions fit your requirements (latency, data residency, or compliance), contact [sales@meilisearch.com](mailto:sales@meilisearch.com) to request a new one.

Reviewer: Maybe it would be a good idea to contact support instead, that way support can direct to sales only the prospects with a potential Enterprise project. Instead of having sales receive all requests for new regions

Author: Changed to contact support@ instead of sales@ for new-region requests.
